themes/bootstrap3/templates/breadcrumbs/multi.phtml

```diff
@@ -17,4 +17,4 @@
 <?php else: ?>
   <li class="active" title="<?=$this->layout()->title ?>"><?=$this->truncate($this->layout()->title, 100) ?></li>
 <?php endif; ?>
-<script>$('.dropdown-toggle').dropdown()</script>
+<?php $this->headScript()->appendScript("$('.dropdown-toggle').dropdown()");
```

Comment:
I suspect that this script is inline for a reason, so it's probably better to use the inline script helper rather than the head script helper. I'm also not sure if this template is actually used, so it may not really matter. Either way, perhaps we should take care of eliminating explicit script tags in a separate pull request, since that's a worthwhile effort independent of CSP implementation, and we can probably get it merged sooner.

Reply:
changed in follow-up